modules/core/src/v2/coins/btc.ts

```diff
@@ -71,27 +71,46 @@ export class Btc extends AbstractUtxoCoin {
     return common.Environments[this.bitgo.getEnv()].blockchairBaseUrl + url;
   }
 
-  getAddressInfoFromExplorer(addressBase58: string): Bluebird<any> {
+  /**
+   * Sanitize api key, decorate it so it is ready to be appended to an URL of an explorer
+   * @param {string} apiKey
+   * @returns {string}
+   */
+  transformExplorersApiKey(apiKey?: string): string { // TODO: make this works with multiple explorers
```

Comment:
Some explorers might require passing the api key in the request header. 

If we want to really support multiple explorer backends we need to do this at a higher level and have an interface like `new SomeExplorer(apiKey).getUnspents(address)`